apps/newsletters-ui/src/app/components/RawDataDialog.tsx

```diff
@@ -74,6 +80,9 @@ export const RawDataDialog = ({ record, title = 'raw data' }: Props) => {
 				</TableContainer>
 				<DialogActions>
 					<Button onClick={copyJson}>copy json</Button>
+					{editHref && (
+						<NavigateButton href={editHref}>edit json</NavigateButton>
```

Comment:
Will it be possible (eventually, in a separate PR) to restrict access to this option by hiding it for users who don't have the required access rights?  If so, maybe add a TODO comment